bindings/python/mata.pyx

```diff
@@ -720,6 +720,36 @@ cdef class Nfa:
 
         return noodles
 
+    @classmethod
+    def noodlify_for_equation(cls, left_side_automata: list[Nfa], Nfa right_side_automaton, include_empty = False):
+        """
+        Create noodles for equation.
```

Comment:
I don't understand what this function should do (from the comment). What is equation? Maybe some short example would be helpful here. Like "For example, the left side automata can represent regular expression ..., and right side represents formula phi = a || b. The result is then automaton representing all words a*" (yes this is nonsense, but some intuition would help me a lot.